### PR TITLE
Added script for removing/updating devices in database

### PR DESCRIPTION
**Pull request by IvanRuzavin**
This workflow will update or remove existing Devices based on provided regex.

### CODE REVIEW

File: scripts/update_devices_db.py [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+                    None,
+                    None,
+                    None,
+                    query
+                    )
+
+
+    return
+
+def removeDeviceFromDb(dbs, regex):
```
Comment (StrahinjaJacimovic) [inline scripts/update_devices_db.py:219]: Add an option to keep the device in the database, just hide it in SDK.

Take care of special cases for XC compilers.

File: scripts/update_devices_db.py [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+    ## Reupload databases over existing assets
+    archive_path = compress_directory_7z(os.path.join(os.path.dirname(__file__), 'databases'), f'{dbPackageName}.7z')
+    async with aiohttp.ClientSession() as session:
+        upload_result = await upload_release_asset(session, token, repo, archive_path, release_version)
+    if databaseErp:
+        async with aiohttp.ClientSession() as session:
+            upload_result = await upload_release_asset(session, token, repo, databaseErp, release_version)
+
+    ## Step 16 - overwrite the existing necto_db.db in root with newly generated one
```
Comment (StrahinjaJacimovic) [inline scripts/update_devices_db.py:261]: Not step 16. :)

File: .github/workflows/updateDbDevices.yaml [hunk 3]
```diff
@@ -0,0 +1,146 @@
+name: Remove/Update Devices in the Database
+
+on:
+  workflow_dispatch:
+    inputs:
```
Comment (StrahinjaJacimovic) [inline .github/workflows/updateDbDevices.yaml:5]: When you add the delete or hide checkbox, i'm thinking set the HIDE as default.

I would rather have the device accidentally hidden then removed from SDK altogether.

**Review by StrahinjaJacimovic — CHANGES_REQUESTED** [review]

File: scripts/update_devices_db.py (hunk 2 above)
Comment (IvanRuzavin) [reply scripts/update_devices_db.py:261]: Removed "Step 16" from the comment.

File: .github/workflows/updateDbDevices.yaml (hunk 3 above)
Comment (IvanRuzavin) [reply .github/workflows/updateDbDevices.yaml:5]: Added a checkbox for this option.

File: scripts/update_devices_db.py (hunk 1 above)
Comment (IvanRuzavin) [reply scripts/update_devices_db.py]: Added a checkbox for XC8 support adding.

**Review by StrahinjaJacimovic — APPROVED** [review]

**Pull request merged**